scipy/stats/stats.py

```diff
@@ -5948,7 +5974,23 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
     ...                 random_state=np.random.default_rng(12345))
     (-1.467966985449, 0.14)
 
+    Take these two samples, one of which has an extreme tail.
+
+    >>> a = (56, 128.6, 12, 123.8, 64.34, 78, 763.3)
+    >>> b = (1.1, 2.9, 4.2)
+
+    Use the `trim` keyword to trim the input samples. Using 20% trimming,
+    `trim=.2`, the test will trim 1 element off of each tail for sample `a`,
+    which is of length 7, and no elements off of sample `b`, because it is
+    of length 3.
+
+    >>> stats.ttest_ind(a, b, trim=.2)
+    Ttest_indResult(statistic=3.4463884028073513,
+                    pvalue=0.01369338726499547)
```

Comment:
This needs to be updated due to the new seeding in examples (just got merged now). You will need to run the doctests with `python runtests.py -g -n --refguide-check` to get the correct seeding.

Reply:
Hi @tupui, The data in this example is not randomly generated, so I don't think this will be an issue:

>     >>> a = (56, 128.6, 12, 123.8, 64.34, 78, 763.3)
>     >>> b = (1.1, 2.9, 4.2)

Reply:
My bad I misread 🙇‍♂️

Reply:
@tupui hope it's OK to close this for you. Looks resolved.

Reply:
Yes thanks Matt.